parsec/core/gui/app.py

```diff
@@ -49,41 +49,48 @@ def _before_quit():
 
 
 async def _start_ipc_server(config, main_window, start_arg, result_queue):
-    new_instance_needed_qt = ThreadSafeQtSignal(main_window, "new_instance_needed", object)
```

Comment:
I think this function should be called `_run_ipc_server` since it's a long running task.

Reply:
:+1: